README.md

```diff
@@ -225,9 +173,7 @@ Example:
 
 > Describe 1 - 3 features you will add to the project.
 
```

Comment:
[OPTIONAL] 
- [ ] please Kindly remove these texts as they only serve descriptive purposes which are to guide you on how to customise this section of your readme file, once customized to your taste as you have done you do not need them anymore and you can take it off :+1: